src/plugins/kconfig/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+include (LibAddMacros)
+
+set (TEST_ARGUMENTS ADD_TEST CPP_TEST INSTALL_TEST_DATA)
```

Comment:
Please add these arguments directly in the `add_pugin` call below. Using a variable just makes the code harder to read.